### PR TITLE
Add dns-query for code review

### PR DESCRIPTION
**Pull request by srsudar**

### CODE REVIEW

File: chromeapp/test/scripts/dnssd/dns-query-test.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+'use strict';
+var test = require('tape');
+var dnsQuery = require('../../../app/scripts/dnssd/dns-query');
+
+test('create a query', function(t) {
+  var name = 'www.example.com';
+  var queryType = 3;
+  var queryClass = 4;
+  // // Corresponds to 155.33.17.68
```
Comment (mdwelsh) [inline chromeapp/test/scripts/dnssd/dns-query-test.js]: Dead code?

File: chromeapp/test/scripts/dnssd/dns-query-test.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+test('serializes and deserializes correctly', function(t) {
+  var domainName = '_semcache._http.local';
+  var queryType = 1;
+  var queryClass = 2;
+
+  var expected = new dnsQuery.DnsQuery(domainName, queryType, queryClass);
+
+  var serialized = expected.serialize();
+
+  var actual = dnsQuery.createQueryFromByteArray(serialized);
```
Comment (mdwelsh) [inline chromeapp/test/scripts/dnssd/dns-query-test.js:29]: Add test where startByte != 0

File: chromeapp/test/scripts/dnssd/dns-query-test.js (hunk 1 above)
Comment (srsudar) [inline chromeapp/test/scripts/dnssd/dns-query-test.js]: Whoops; removed.

Done.

File: chromeapp/test/scripts/dnssd/dns-query-test.js (hunk 2 above)
Comment (srsudar) [inline chromeapp/test/scripts/dnssd/dns-query-test.js:29]: done

**Pull request merged**